cursive/custom_types.py

```diff
@@ -58,21 +61,24 @@ def __init__(self, **data):
         if not self.id:
             self.id = random_id()
 
+
 class CursiveSetupOptionsExpand(BaseModel):
     enabled: Optional[bool] = None
     defaults_to: Optional[str] = None
-    model_mapping: Optional[dict[str, str]] = None
+    resolve_model: Optional[dict[str, str]] = None
```

Comment:
I believe `model_mapping` is more descriptive, and changing this breaks the interface (it would require a major version release, breaking compatibility).

Reply:
Totally, I tousled with this and went back and forth. 

Using model_mapping prints a warning with PydanticV2 though :( 

Another descriptive name would be resolve_model_alias...